backend/tracim_backend/lib/mail_notifier/notifier.py

```diff
@@ -20,6 +20,7 @@
 from tracim_backend.lib.mail_notifier.utils import EST
 from tracim_backend.lib.mail_notifier.utils import SmtpConfiguration
 from tracim_backend.lib.utils.logger import logger
+from tracim_backend.lib.utils.logger import notification_logger
```

Comment:
I suggest to move the notification_logger to `tracim_backend/lib/mail_notifier`. what do you think? This logger is more "functionnal" than "technical". What do you think?

Reply:
That's a correct choice as soon as we don't want to log another type of notifcation than email one.
I'm thinking about this logger as more generic than just a "email notification logger".

Reply:
@lebouquetin so should we consider the notifier as email notifier or should we be more global ?
I do think if we decide to consider it as email-only, that we should change the naming.

Reply:
as we talked, move the logger to mail module